cfgmgr/buffermgrdyn.cpp

```diff
@@ -910,10 +910,8 @@ task_process_status BufferMgrDynamic::doUpdateStaticProfileTask(buffer_profile_t
     return task_process_status::task_success;
 }
 
-task_process_status BufferMgrDynamic::handleBufferMaxParam(Consumer &consumer)
+task_process_status BufferMgrDynamic::handleBufferMaxParam(KeyOpFieldsValuesTuple &t)
```

Comment:
Please rename all occurrences of KeyOpFieldsValuesTuple 't' to 'tuple' so that it is more meaningful